src/infrastructure/notify.rs

```diff
@@ -97,7 +97,8 @@ pub fn user_registered(user: &User, url: &str) {
 
 pub fn user_password_reset_requested(token: &EmailToken) {
     let url = format!(
-        "https://kartevonmorgen.org/#/?reset_password={}",
+        //TODO: "https://kartevonmorgen.org/#/?reset_password={}",
+        "https://openfairdb.org/password_reset?token={}",
```

Comment:
"password_reset" or "password-reset" or "reset-passwort"?

```
#[get("/reset-password?<token>")]
pub fn get_password_reset(flash: Option<FlashMessage>, token: &RawStr) -> Markup {
    view::password_reset(flash, "/users/actions/reset-password", token)
}
```

Very confusing to follow all dependencies and links without a clear naming convention, I'm totally lost.

Reply:
You're totally right! I'm going to change that :)